bundler/lib/bundler/cli/doctor.rb

```diff
@@ -115,7 +115,7 @@ def check_home_permissions
       Find.find(Bundler.bundle_path.to_s).each do |f|
         if !File.exist?(f)
           broken_symlinks << f
-        elsif !File.writable?(f) || !File.readable?(f)
+        elsif !File.readable?(f)
           if File.stat(f).uid != Process.uid
             files_not_rw_and_owned_by_different_user << f
```

Comment:
@Edouard-chin We should probably rename these local variables now to not mention "rw" or "writable".

Reply:
👍 Done !